pkg/kubelet/eviction/helpers.go

```diff
@@ -364,6 +366,16 @@ func parseThresholdStatements(statements map[string]string) ([]evictionapi.Thres
 			results = append(results, *result)
 		}
 	}
+	if utilfeature.DefaultFeatureGate.Enabled(features.InheritDefaultEvictionValues) {
```

Comment:
if len(statements) == 0, we never get here... is that expected?

Reply:
Yes, it is expected. if len(statements) == 0, this case means that the user doesn't have modified the threshold values. In that case all the parameters of eviction hard threshold will get their default values and it is what we want. In this case, old behaviour is perfect.

Reply:
I don't think that's accurate... `evictionHard: {}` is not nil, so will not get defaulted, but is length 0, so will not reach this branch

In fact, `evictionHard: {}` is how the user currently disables kubelet eviction based on these signals, and this PR makes that impossible